packages/core/src/modules/mdoc/MdocOptions.ts

```diff
@@ -0,0 +1,58 @@
+import type { Mdoc } from './Mdoc'
+import type { Key } from '../../crypto/Key'
+import type { DifPresentationExchangeDefinition } from '../dif-presentation-exchange'
+import type { ValidityInfo, MdocNameSpaces } from '@protokoll/mdoc-client'
+
+export type { MdocNameSpaces } from '@protokoll/mdoc-client'
+
+export interface MdocVerificationContext {
```

Comment:
Can't we reuse global other verification context for this?

Reply:
For the w3cCredentials the context looks like this. Or what global context do you mean?

export interface VerificationContext {
  /**
   * The `id` of the `ProofRecord` that this verification is bound to.
   */
  didcommProofRecordId?: string

  /**
   * The `id` of the `OpenId4VcVerificationSessionRecord` that this verification is bound to.
   */
  openId4VcVerificationSessionId?: string
}